test/Microsoft.Health.Dicom.Tests.Integration/Persistence/ExternalFileStoreTests.cs

```diff
@@ -146,6 +146,9 @@ public async Task GivenFileWithETag_WhenOperatingOnFileWithDifferentETagForCondi
 
         var copyFileEx = await Assert.ThrowsAsync<DataStoreRequestFailedException>(() => _blobDataStore.CopyFileAsync(version, _getNextWatermark(), Partition.Default, badFileProperties));
         Assert.Contains(ExpectedCopyFailedSubstring(), copyFileEx.Message);
+
+        var getFilePropsEx = await Assert.ThrowsAsync<DataStoreRequestFailedException>(() => _blobDataStore.GetFilePropertiesAsync(version, Partition.Default, badFileProperties));
+        Assert.Contains(ExpectedCopyFailedSubstring(), getFilePropsEx.Message);
```

Comment:
The method name `ExpectedCopyFailedSubstring()` seems inaccurate. Can you change it?